lib/plugin/instance.rb

```diff
@@ -849,6 +849,12 @@ def enabled_site_setting(setting = nil)
     end
   end
 
+  # Register a new valid site setting area.
+  # Site setting area is a way to group site settings.
```

Comment:
I feel like this could use some slight elaboration:

> Site setting areas are a way to group site settings below
> the setting category level. This is useful for creating focused
> config areas that update a small selection of settings, and otherwise
> grouping related settings in the UI.